iml-gui/crate/src/lib.rs

```diff
@@ -5,10 +5,14 @@
 #![allow(clippy::non_ascii_literal)]
 #![allow(clippy::enum_glob_use)]
 
+#[macro_use]
+extern crate seed;
```

Comment:
This is to enable `log!` macro from `seed` framework

Reply:
With the 2018 edition you can bring macros into scope as needed instead of importing them globally.

In order to use this, you can just do `seed::log!(...)` wherever you want to log.

Reply:
Yeah, indeed.